tests/legacy/ExpectedProfileFailures.py

```diff
@@ -12,12 +12,12 @@ def sleep():
 
 def wait_for_test(stimulus, wait_time):
     test_state = stimulus.GetStimulusProfileManagerState()
-    start_time = time.clock()
+    start_time = time.process_time()
```

Comment:
I think here would be better to also use `perf_counter()` to account for sleeps, not just CPU time.